build/cli.js

```diff
@@ -149,16 +148,14 @@ async function buildDocuments(
       // mutating the built document which makes this not great and refactor-worthy.
       JSON.stringify({ doc: builtDocument })
     );
-    // There are some archived documents that, due to possible corruption or other
+    // There are some documents that, due to possible corruption or other
     // unknown reasons, don't have a list of contributors.
-    if (document.metadata.contributors || !document.isArchive) {
+    if (document.metadata.contributors) {
```

Comment:
Based on the original logic, this conditional code would **always** be run for non-archived content since `!document.isArchive` would always be true for non-archived content. Does that mean we really want to make this conditional code unconditional, so run it all the time?